### ISSUE TITLE
feat: Crear modelo base de clasificación

### ISSUE THREAD

**Issue by hdeavila1992** [hdeavila1992/Robot_Human_Interfase_v0#1]
El objetivo de esta tarea es entrenar un primer modelo simple (baseline) que pueda clasificar las señales EMG en los 6 estados definidos, utilizando el conjunto de datos de características ya procesado.

**Estos pueden ser los pasos:**

1. Cargar el dataset de características desde data/processed/final_feature_data.parquet.
2. Separar los datos en la matriz de características (X) y el vector de etiquetas (y).
3. Codificar las etiquetas (y) de texto a números usando sklearn.preprocessing.LabelEncoder.
4. Dividir el dataset en conjuntos de entrenamiento y prueba (sugiero 80% para entrenamiento y 20% para prueba) usando sklearn.model_selection.train_test_split.
5. Entrenar un modelo simple de clasificación. Una Regresión Logística (sklearn.linear_model.LogisticRegression) es un excelente punto de partida.
6. Evaluar el rendimiento del modelo en el conjunto de prueba. Como mínimo, se debe imprimir el accuracy y una matriz de confusión.

### **Criterios de Aceptación**
La tarea se considerará completa cuando:

1. Exista un nuevo notebook en la carpeta notebooks/, por ejemplo 03-modeling-baseline.ipynb, que contenga todo el proceso.
2. El notebook se ejecute sin errores y muestre el resultado de la evaluación del modelo.

**Comment by hdeavila1992** [comment]
Parece ser que hay un problema de interpretación de los datos, procederé entonces a buscar referencias bibliográficas relacionadas con el tema. @JheiferPaez
